### PR TITLE
Add tokens for EVM blockchains

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/acalaevm/assets/0x54A37A01cD75B616D63E0ab665bFfdb0143c52AE/info.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{    
+"name": "Dai Stablecoin",
```
Comment (vcoolish) [inline blockchains/acalaevm/assets/0x54A37A01cD75B616D63E0ab665bFfdb0143c52AE/info.json:2]: formatting

File: blockchains/klaytn/assets/0x6270B58BE569a7c0b8f47594F191631Ae5b2C86C/info.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+"name": "USD Coin",
+"website": "https://centre.io/usdc",
+"description": "USDC is a fully collateralized US dollar stablecoin, an Ethereum powered coin and is the brainchild of CENTRE, an open source project bootstrapped by contributions from Circle and Coinbase.",
+"explorer": "https://scope.klaytn.com/token/0x6270b58be569a7c0b8f47594f191631ae5b2c86c",
+"type": "KlAYTN",
```
Comment (vcoolish) [inline blockchains/klaytn/assets/0x6270B58BE569a7c0b8f47594F191631Ae5b2C86C/info.json:6]: formatting and KLAYTN

File: blockchains/klaytn/assets/0x5c74070FDeA071359b86082bd9f9b3dEaafbe32b/info.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{    
+"name": "Klaytn Dai",
+"website": "http://makerdao.com",
+"description": "Dai is a stable cryptocurrency supported by Maker (MKR). To ensure price stability, Dai minimizes the price volatility against the US dollar, through an incentive structure for its participants.",
+"explorer": "https://scope.klaytn.com/token/0x5c74070fdea071359b86082bd9f9b3deaafbe32b",
```
Comment (vcoolish) [inline blockchains/klaytn/assets/0x5c74070FDeA071359b86082bd9f9b3dEaafbe32b/info.json:5]: formatting

File: blockchains/confluxespace/assets/0xa47f43DE2f9623aCb395CA4905746496D2014d57/info.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+"name": "Ethereum",
+"website": "https://ethereum.org/",
+"description": "Ethereum is a global, open-source platform for decentralized applications. Eth is fueling transactions on the Chain.",
```
Comment (vcoolish) [inline blockchains/confluxespace/assets/0xa47f43DE2f9623aCb395CA4905746496D2014d57/info.json:4]: formatting

**Review by defisaur — APPROVED** [review]

**Pull request merged**